Jellyfin/Controls/JellyfinWebView.xaml

```diff
@@ -2,12 +2,11 @@
     x:Class="Jellyfin.Controls.JellyfinWebView"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:jellyfin_uwp.Controls"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
```

Comment:
This breaks controller support on xbox and pc

Reply:
adding the `xmlns:local="using:jellyfin_uwp.Controls"` will fix that? I have neither a controller or an xbox to test with.